server/lib/configuration.js

```diff
@@ -451,6 +451,13 @@ const conf = module.exports = convict({
     }
   },
   sms: {
+    redirect: {
+      targetLink: {
+        default: 'https://app.adjust.com/2uo1qc?campaign=fxa-conf-page&adgroup=android&creative=button',
```

Comment:
The current URL used by the auth-server redirects to:

https://app.adjust.com/2uo1qc?campaign=fxa-conf-page&adgroup=sms&creative=link, update this.